proxy/src/main/java/org/apache/rocketmq/proxy/remoting/RemotingProtocolServer.java

```diff
@@ -258,6 +259,10 @@ protected RequestPipeline createRequestPipeline() {
         };
 
         List<AccessValidator> accessValidatorList = new ArrayList<>();
+        ProxyConfig config = ConfigurationManager.getProxyConfig();
+        if (config.isEnableACL()) {
```

Comment:
May the if judgment be removed, for the same reason

Reply:
I think this judgment should exist because if EnableACL is false, the list should be empty

Reply:
If enableACL is false and then updated to true, then acl is not active because list is empty

Reply:
alright